README.md

```diff
@@ -83,14 +83,14 @@ This file may be good for experienced developers who:
   <tr>
     <th>Technology</th>
     <th>Yandex internal</th>
-    <th>Analogs</th>
+    <Th>Analogs</th>
```

Comment:
Accidental uppercase? Thanks for the rest.

Reply:
fixed